doc/start/os-recommendations.rst

```diff
@@ -78,6 +78,24 @@ non-Linux systems but these are not supported by the core Ceph effort.
    
    ``Btrfs`` is no longer tested on Centos 7 in the Octopus release. We recommend using ``bluestore`` instead.
 
+Tested Container Hosts
+======================
+
+This is a list of all operating systems that have been tested as container hosts.
+
++---------------+---------------------------------+
+| OS            | Tested as container host?       |
++===============+=================================+
+| Centos 9      | yes                             |
++---------------+---------------------------------+
+| Ubuntu 22.04  | yes                             |
```

Comment:
the container host distros will differ between ceph releases. since we already have a table of distros vs. releases above, what do you think about adding a new letter to that?

`**H**: Ceph tests this distro as a container host.`

Reply:
> the container host distros will differ between ceph releases. since we already have a table of distros vs. releases above, what do you think about adding a new letter to that?
> 
> 
> 
> `**H**: Ceph tests this distro as a container host.`

@cbodley, I considered that and then went the opposite way while wondering if someone would suggest precisely what you have suggested. However, now that two minds have independently conceived of the idea and one of them connected to fresh eyes has expressed a preference for it, we're going to do it your one-chart way.

Reply:
I think that I've added ``H`` to all the cells in the chart that need some ``H``. Let me know if I've missed any.